tests/test_sanity.py

```diff
@@ -38,6 +39,7 @@ def test_java_from_host(
         java_safemode=False,
         java_mode="ap",
     ) as profiler:
+        _ = assert_application_name  # Fixture only for asserting application name upon test teardown.
```

Comment:
Why is this line required?

Reply:
Avoid mypy "unused variable" warning. It's a common idiom to force a fixture to be used

Reply:
Maybe could make the comment more descriptive in this regard

Reply:
Done

Reply:
@guybortnikov I suggest replacing this comment with something like `# Silence mypy - this fixture asserts application name upon test teardown`.

AFAIC this is my last comment on the PR.

Reply:
Oh I just saw the change. NVM, I approve.